buttercup/cogs/rules.py

```diff
@@ -0,0 +1,95 @@
+from datetime import datetime
+
+import asyncpraw
+from asyncprawcore import NotFound, Redirect
+from discord import Embed
+from discord.ext.commands import Cog
+from discord_slash import SlashContext, cog_ext
+from discord_slash.utils.manage_commands import create_option
+
+from buttercup.bot import ButtercupBot
+from buttercup.strings import translation
+
+i18n = translation()
+
+
+def extract_sub_name(subreddit: str) -> str:
+    """Extract the name of the sub without prefix."""
+    if subreddit.startswith("/r/"):
+        return subreddit[3:]
+    if subreddit.startswith("r/"):
+        return subreddit[2:]
+    return subreddit
+
+
+class Rules(Cog):
+    def __init__(self, bot: ButtercupBot, reddit_api: asyncpraw.Reddit) -> None:
+        """Initialize the Rules cog."""
+        self.bot = bot
+        self.reddit_api = reddit_api
+
+    @cog_ext.cog_slash(
+        name="rules",
+        description="Get the rules of the specified subreddit.",
+        options=[
+            create_option(
+                name="subreddit",
+                description="The subreddit to get the rules of.",

```

Comment:
It might be worth to mention here that the `/r/` prefix is not to be included

Reply:
It should work with `/r/` as well! (I think) 😆

Reply:
I just tested again, and it works